src/app/index.js

```diff
@@ -699,6 +699,25 @@ class AccessibilityCheckerHighlight {
 	 * This function disables all styles on the page.
 	 */
 	disableStyles() {
+
+		/*
+		If the site compiles css into a combined file, our method for disabling styles will cause out app's css to break.
+		This checks if the app's css is loading into #edac-app-css as expected. 
+		If not, then we assume the css has been combined, so we manually add it to the document.
+		*/
+		if( ! document.querySelector('#edac-app-css') ){
+			debug('css is combined, so adding app.css to page.');
```

Comment:
Does the build script remove debug calls?

Reply:
It doesn't. The  debug and info functions just check for  INFO_ENABLED & DEBUG_ENABLED flag variables before they output anything to console. I think uglify removes console and comments though if you want to go that route?

Reply:
It's not necessary right now. Maybe something to think about in the future. There might be a performance benefit in doing so.